### PR TITLE
fix #82161 ndtv.com

### LINKED ISSUE #82161: ndtv.com
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***: https://www.ndtv.com/education/aktu-releases-revised-odd-semester-exams-dates has Advertisement leftovers
image

image

### PR DESCRIPTION
**Pull request by wout-codes**
#82161

### CODE REVIEW

File: SpywareFilter/sections/general_url.txt [hunk 1]
```diff
@@ -98,6 +98,7 @@ directtrack.*.js
 /watch.*.js?key=*&kw=*&refer=
 /pv_count.
 _analytics.js?
+/static/js/analytics-helper.js
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/general_url.txt:101]: I don't see this request, btw it doesn't look popular, so it's better to add to specific with $domain modifier
(or remove this rule if this request does not exist anymore)

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/general_url.txt (hunk 1 above)
Comment (wout-codes) [reply SpywareFilter/sections/general_url.txt:101]: Let's take risk. The tracking protection filter is so outdated

Comment (wout-codes) [reply SpywareFilter/sections/general_url.txt:101]: EasyPrivacy beats it

Comment (Sergey-Lyapin) [reply SpywareFilter/sections/general_url.txt:101]: @Alex-302 please decide

Comment (Alex-302) [reply SpywareFilter/sections/general_url.txt:101]: https://www.ndtv.com/static/js/analytics-helper.js

Comment (Alex-302) [reply SpywareFilter/sections/general_url.txt:101]: But it is just google analytics, low priority issue.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**